pkg/k8s/watchers/pod.go

```diff
@@ -696,6 +696,12 @@ func (k *K8sPodWatcher) genServiceMappings(pod *slim_corev1.Pod, podIPs []string
 			var nodeAddrAll []netip.Addr
 			loopbackHostport := false
 
+			hostIP := net.ParseIP(pod.Status.HostIP)
+			if hostIP == nil {
+				logger.Warningf("Skip creating HostPort service, no/invalid pod HostIP.")
```

Comment:
Minor nits here:

```suggestion
				logger.WithFields(logfields.IPAddr, pod.Status.HostIP.Warning("Skip creating HostPort service, no/invalid pod HostIP.")
```

Reply:
We parse `pod.HostIP` above; is there a difference between `pod.HostIP` and `pod.Status.HostIP`?

Reply:
I  have used debug messages with logfields instead of warning messages. `pod.Status.HostIP` is already included in the logfields.